atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -161,8 +161,8 @@ private void cas(Client client, Long oldVal, long newVal) {
     }
 
     private void logError(String msg, Throwable cause) {
-        log.error(msg, cause);
-        DebugLogger.logger.error(msg, cause);
+        log.error("Error: {}", msg, cause);
+        DebugLogger.logger.error("Error: {}", msg, cause);
```

Comment:
The logError method is only used in 2 places so we could inline it.  Or maybe we can pass the template and arguemnts to this method using varargs so everything looks ok to this method? (not sure if this works)